core/src/main/java/com/novoda/noplayer/exoplayer/TrackType.java

```diff
@@ -0,0 +1,7 @@
+package com.novoda.noplayer.exoplayer;
+
+public enum TrackType {
```

Comment:
Yes. If we move the `RendererTrackIndexExtractor` to the `mediasource` package, along with this, we should be able to make this package.